### DISCUSSION TITLE
v3.3.2 Space Between Not Working

### DISCUSSION THREAD

**Discussion by bnuby** [tailwindlabs/tailwindcss#11585] [Help]
**What version of Tailwind CSS are you using?**
v3.3.2
https://registry.yarnpkg.com/tailwindcss/-/tailwindcss-3.3.2.tgz#2f9e35d715fdf0bbf674d90147a0684d7054a2d3

**What build tool (or framework if it abstracts the build tool) are you using?**
Next.js 13.0.2

**What version of Node.js are you using?**
v18.16.1

**What browser are you using?**
Chrome

**What operating system are you using?**
macOS

The space between class not working after build.

found that the class selectors are different in development and after production build 

Development: (from Chrome inspector)
![image](https://github.com/tailwindlabs/tailwindcss/assets/22321340/815457df-e326-4fd4-8095-3fa3b554e22a)

Production:
![image](https://github.com/tailwindlabs/tailwindcss/assets/22321340/1dacdfee-a466-4cba-8c87-bddf0d114137)

tailwind.config.js
```js=
/** @type {import('tailwindcss').Config} */
module.exports = {
  important: '#app',
  content: [
    './src/**/**/*.{ts,tsx}',
    './node_modules/react-tailwindcss-datepicker/dist/index.esm.js',
  ],
  theme: {
    container: {
      center: true,
    },
    colors: {
      inherit: 'inherit',
      current: 'currentColor',
      transparent: 'transparent',
      'primary-shadow': 'hsl(var(--primary-shadow) / <alpha-value>)',
      'primary-content-faded': '#BFBFBF',
      failure: 'var(--failure)',
      'failure-content': 'var(--failure-content)',
    },
    fontSize: {
      xss: ['0.625rem', { lineHeight: '1.125rem' }],
      xs: ['0.75rem', { lineHeight: '1.25rem' }],
      sm: ['0.875rem', { lineHeight: '1.375rem' }],
      base: ['1rem', { lineHeight: '1.5rem' }],
      lg: ['1.125rem', { lineHeight: '1.625rem' }],
      xl: ['1.25rem', { lineHeight: '1.75rem' }],
      '2xl': ['1.375rem', { lineHeight: '1.875rem' }],
      '3xl': ['1.5rem', { lineHeight: '2rem' }],
      '4xl': ['1.75rem', { lineHeight: '2.25rem' }],
      '5xl': ['2rem', { lineHeight: '2.5rem' }],
    },
    fontFamily: {},
    extend: {
      boxShadow: {
        DEFAULT: '0 18px 28px rgb(0 0 0 / 0.1)',
        none: 'none',
        card: '0px 8px 12px rgba(9, 30, 66, 0.07)',
      },
      zIndex: {
        'nav-bar': '12',
      },
    },
  },
  daisyui: {
    logs: false,
    themes: [
      {
        light: {
          // Other colors
        },
      },
      {
        dark: {
          ...require('daisyui/src/colors/themes')['[data-theme=dark]'],
          // Other colors
        },
      },
    ],
  },
  plugins: [require('@tailwindcss/typography'), require('daisyui'), 
  // Created custom plugin
  require('./plugins/dropdown.config'), require('./plugins/button.config')],
};
```

global.css
<img width="402" alt="image" src="https://github.com/tailwindlabs/tailwindcss/assets/22321340/e6746670-1380-4504-be33-9e105d71143e">

**Comment by RobinMalfait** [comment]
Hey, thank you for this bug report! 🙏

Tailwind CSS itself doesn't have a difference between "development" or "production" mode. This is likely a side effect of other PostCSS plugins that are applied conditionally or it can also be caused by other steps if you are integration Tailwind CSS in an existing build-chain.

Can you share a reproduction repo so that we can take a look?

**Reply by bnuby** [reply]
Thank you! 
Your comment help me a lots.

It only happened when I use **important** config in tailwind config

Seems like theres a compatible issue between next and **tailwindcss**.
After I upgrade my next package, than the problem is gone.
